login.html

```diff
@@ -117,8 +117,8 @@
   function submitForm() {
```

Comment:
I suggest refactoring this to take an action, and success/failure callbacks as parameters. Avoid code duplication.